### PR TITLE
Add missing WC Admin function stubs

### PR DESCRIPTION
**Pull request by the-csaba**
Hi there

WC Admin functions from `includes` dir.

Thanks!

### CODE REVIEW

File: finder-packages.php [hunk 1]
```diff
@@ -6,6 +6,7 @@
         'source/woocommerce/packages/action-scheduler/classes',
```
Comment (szepeviktor) [inline finder-packages.php]: could you move the new line here
now it looks line the comment is also for that line

File: finder-packages.php [hunk 2]
```diff
@@ -16,5 +17,9 @@
     )
     // Uses WP-CLI.
     ->notPath('WP_CLI')
+    // Exclude email templates.
+    ->notPath('emails')
+    // Update functions.
+    ->notPath('wc-admin-update-functions.php')
```
Comment (szepeviktor) [inline finder-packages.php]: These are too general, I mean valid for other `in`-s.
Could we have 1 ->append instead?

File: finder-packages.php (hunk 1 above)
Comment (the-csaba) [reply finder-packages.php]: Yes of course

**Review by szepeviktor — APPROVED** [review]

**Comment by szepeviktor** [comment]
Thank you.

**Comment by the-csaba** [comment]
Thanks for merging!

**Pull request merged**